src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -38,13 +38,13 @@ public void testIsFull() {
 		}
 
 		boolean result = ryhmer.isFull();
-		Assert.assertEquals(true, result);
+		Assert.assertEquals(false, result);
 	}
 
 	@Test
 	public void testPeekaboo() {
 		DefaultCountingOutRhymer ryhmer = new DefaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
+		final int EMPTY_STACK_VALUE = 0;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).